submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListSubmissionTest.kt

```diff
@@ -179,34 +181,86 @@ internal class FileListSubmissionTest(private val tempFolder: TemporaryFolder) :
-                    NfsFile("$fileListName.xml", File(subFolder).resolve("Files/$fileListName.xml")),
-                    NfsFile("$fileListName.pagetab.tsv", File(subFolder).resolve("Files/$fileListName.pagetab.tsv"))
-                )
-            } else listOf()
-
-            val submissionPageTabFiles = if (mongoMode) {
-                if (enableFire) listOf() else listOf(
-                    NfsFile("$accNo.json", File(subFolder).resolve("$accNo.json")),
-                    NfsFile("$accNo.xml", File(subFolder).resolve("$accNo.xml")),
-                    NfsFile("$accNo.pagetab.tsv", File(subFolder).resolve("$accNo.pagetab.tsv"))
-                )
-            } else listOf()
-
-            assertThat(createdSub.section.fileList).isEqualTo(ExtFileList("FileList", pageTabFiles = expectedTabFiles))
-            assertThat((createdSub.pageTabFiles)).isEqualTo(submissionPageTabFiles)
+            if (mongoMode)
+                if (enableFire) {
+                    val submissionTabFiles = createdSub.pageTabFiles as List<FireFile>
+                    assertThat(submissionTabFiles).hasSize(3)
+                    assertThat(submissionTabFiles).isEqualTo(submissionFireTabFiles(accNo, subFolder))
+
+                    val fileListTabFiles = createdSub.section.fileList!!.pageTabFiles as List<FireFile>
+                    assertThat(fileListTabFiles).hasSize(3)
+                    assertThat(fileListTabFiles).isEqualTo(fileListFireTabFiles(subFolder))
+                } else {
+                    val submissionTabFiles = createdSub.pageTabFiles as List<NfsFile>
+                    assertThat(submissionTabFiles).hasSize(3)
+                    assertThat(submissionTabFiles).isEqualTo(submissionNfsTabFiles(accNo, subFolder))
+
+                    val fileListTabFiles = createdSub.section.fileList!!.pageTabFiles as List<NfsFile>
+                    assertThat(fileListTabFiles).hasSize(3)
+                    assertThat(fileListTabFiles).isEqualTo(fileListNfsTabFiles(subFolder))
+                }
 
             assertThat(Paths.get("$subFolder/Files/$testFile")).exists()
-            assertThat(Paths.get("$subFolder/Files/$fileListName.xml")).exists()
-            assertThat(Paths.get("$subFolder/Files/$fileListName.json")).exists()
-            assertThat(Paths.get("$subFolder/Files/$fileListName.pagetab.tsv")).exists()
+            assertThat(Paths.get("$subFolder/Files/FileList.xml")).exists()
+            assertThat(Paths.get("$subFolder/Files/FileList.json")).exists()
+            assertThat(Paths.get("$subFolder/Files/FileList.pagetab.tsv")).exists()
 
             assertThat(Paths.get("$subFolder/${createdSub.accNo}.xml")).exists()
             assertThat(Paths.get("$subFolder/${createdSub.accNo}.json")).exists()
             assertThat(Paths.get("$subFolder/${createdSub.accNo}.pagetab.tsv")).exists()
         }
+
+        private fun submissionFireTabFiles(accNo: String, subFolder: String): List<FireFile> {
```

Comment:
For consistency, I think we should always use PageTabFiles rather than TabFiles i.e. `submissionFirePageTabFiles`. The same apply to the other methods.